index.ts

```diff
@@ -48,7 +49,7 @@ function sendMetrics() {
     webVitals: collectedMetrics,
   });
 
-  if (navigator.sendBeacon) {
+  if (typeof window !== 'undefined' && navigator.sendBeacon) {
```

Comment:
Could we do something like `if (window?.navigator.sendBeacon)`?

Reply:
for some reason, it says that the output will always be true, and its not accepting it.

Reply:
alright, whatever 😁